From bbc2610db41b7171c763e24c75fce5b4cd78cfb5 Mon Sep 17 00:00:00 2001
From: Rene Engelhard <rene@debian.org>
Date: Fri, 3 Apr 2020 13:45:49 +0200
Subject: [PATCH] tdf#131835: don't set PYTHONWARNINGS=error if --enable-werror
 isn't set

python 3.8 at least throws DeprecationWarnings for imp->importlib and some
FutureWarnings.
Allow to disable as for the build warnings-are-errors.

Change-Id: Id5ada39f960f014108f2584e3e333741f6345acd
---
 solenv/gbuild/UITest.mk | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/solenv/gbuild/UITest.mk b/solenv/gbuild/UITest.mk
index e428dbd03c48..5b5c6d47298e 100644
--- a/solenv/gbuild/UITest.mk
+++ b/solenv/gbuild/UITest.mk
@@ -73,7 +73,7 @@ else
 		PYTHONPATH="$(PYPATH)" \
 		TestUserDir="$(call gb_Helper_make_url,$(dir $(call gb_UITest_get_target,$*)))" \
 		PYTHONDONTWRITEBYTECODE=0 \
-		PYTHONWARNINGS=error \
+		$(if $(ENABLE_WERROR),PYTHONWARNINGS=error) \
 		$(gb_TEST_ENV_VARS) \
 		$(gb_UITest_COMMAND) \
 		--soffice="$(gb_UITest_SOFFICEARG)" \
-- 
2.20.1

